client_admin/pulp/client/admin/tasks.py

```diff
@@ -31,6 +32,15 @@ class BaseTasksSection(PulpCliSection):
     to provide consistent functionality for a subset of tasks.
     """
 
+    all_flag = PulpCliFlag('--all',  _('if specified, all tasks in all states are shown'),
+                           aliases=['-a'])
+    state_option = PulpCliOption('--state',
+                                 _('comma-separated list of tasks states desired to be '
+                                   'shown. Example: "runnining,waiting,canceled,successful,failed". '
```

Comment:
yeah, 'canceled' is right in this case
`$ pulp-admin -u admin -p admin tasks list --state canceled

+----------------------------------------------------------------------+
                                 Tasks
+----------------------------------------------------------------------+

Operations:  sync
Resources:   pup (repository)
State:       Cancelled
Start Time:  2015-04-22T13:16:32Z
Finish Time: 2015-04-22T13:16:50Z
Task Id:     d0ca6865-91dd-4fbe-ad17-f77db3c61a94

$ pulp-admin -u admin -p admin tasks list --state cancelled

+----------------------------------------------------------------------+
                                 Tasks
+----------------------------------------------------------------------+

No tasks found`